yt_dlp/extractor/vimeo.py

```diff
@@ -212,6 +212,14 @@ def _parse_config(self, config, video_id):
         owner = video_data.get('owner') or {}
         video_uploader_url = owner.get('url')
 
+        audio_quality_map = ('low', 'medium', 'high')
+        for f in formats:
+            if f.get('quality') is None:
+                for key in audio_quality_map:
+                    if f'audio-{key}' in f['format_id']:
+                        f['quality'] = audio_quality_map.index(key) + 1
+                        break
```

Comment:
Why not reverse the check?
extract the `key` from what's present in `f['format_id']` and then check if it exists in the list.

`audio_quality_map.index(f['format_id'].split('-')[-1]) + 1`

Reply:
because there is no guarantee the format_id is `xxxx-audio-{quality}` (actually, it usually is not, but something like `hls-akfire_interconnect_quic-audio-{quality}-{language}`).

I can do the reverse and drop optional language part, but I feel like this way is more future-proof.